platform/testing/contract-testing/how-to-use-pact-for-contract-testing.md

```diff
@@ -85,6 +86,10 @@ PRs related to Pacts will go through the standard [code review process](https://
 
 The process is a collaborative effort where front-end (FE) and back-end (BE) engineers should communicate and may need to iterate on the pacts to get them successfully verified.
 
+#### Merge workflow
+
+Pact tests must be merged in the correct order in order for tests to pass on CI. The provider tests in the back-end must be merged first; contracts will fail until the API endpoint is supported on the provider side. Once the provider test is merged, the consumer test for the front-end can be implemented and merged. Provider states have to be implemented first for the consumer test to pass Pact verification.
```

Comment:
This makes sense. Do you think there'd be any value in adding an ordered list of the workflow to accompany this paragraph?

For example:

1. Implement provider state
2. Merge provider tests in the back-end
3. Implement front-end consumer test
4. Merge front-end consumer test

Reply:
Sure, I like this idea. Thanks for reviewing!